environment/Dockerfile

```diff
@@ -2,8 +2,10 @@ FROM golang:1.13.5
 
 RUN mkdir -p /go/src/GoFire
 
+RUN export GOFIRE_MASTER_HOST=`/sbin/ip route|awk '/default/ { print $3 }'` && export GOFIRE_MASTER=true
+
 ADD . /go/src/GoFire
 
-WORKDIR /go/src/GoFire
+WORKDIR /go/src/GoFire/src
 
-RUN go get -v
\ No newline at end of file
+RUN ls
```

Comment:
We can get rid of this

Reply:
Aye, this is still wip and that's a debug statement